prep-html-skeleton/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Skeleton Exercise</title>
```

Comment:
Hi Mohammed, be sure to put a closing `</head>` tag before your opening `<body>` tag.